test/integration/gssp-redirect/test/index.test.js

```diff
@@ -123,6 +123,10 @@ const runTests = () => {
       window.next.router.push('/gssp-blog/redirect-dest-_another')
     })()`)
     await browser.waitForElementByCss('#another')
+
+    const text = await browser.elementByCss('#another').text()
+
+    expect(text).toEqual('another Page')
```

Comment:
In first commit, I added them to the existing test cases because there are no any "expect" functions.